src/functions-and-arrays.js

```diff
@@ -52,7 +118,24 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+
+function uniquifyArray() {
```

Comment:
Here you had the same issue as before, you were not adding a parameter to your function